Firestore/core/src/firebase/firestore/core/firestore_client.h

```diff
@@ -55,11 +54,19 @@ OBJC_CLASS(FSTSyncEngine);
 
 namespace firebase {
 namespace firestore {
+namespace local {
+
+class Persistence;
+class LruDelegate;
+
+}  // namespace local
 
 namespace remote {
+
 // Forward declaration to make CMAKE build pass as it does not support
```

Comment:
Looks like this comment will become stale pretty soon.

Reply:
The comment is stale, but there's no harm in forward declaring (and indeed some benefit, in reduced build times).

Removed.